Sources/AltSwiftUI/Source/Views/Controls/TextField.swift

```diff
@@ -114,6 +114,7 @@ extension TextField: Renderable {
             view.textAlignment = textAlignment.nsTextAlignment
         }
         view.font = context.viewValues?.font?.font
+        view.adjustsFontForContentSizeCategory = true
```

Comment:
since you are always setting true you can move this code to `createView` instead

Reply:
Thanks. I have just put this statement into `createView`